parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -507,6 +504,7 @@ private synchronized void persistQueue() {
      *
      * @return The stored queue of events.
      */
+    @NonNull
     private ArrayList<Map<String, Object>> getStoredQueue() {
```

Comment:
`getStoredQueue` could never return null. Look lines 525-528. Also, IDE was smart enough to infer it

<img width="579" alt="Screenshot 2023-09-12 at 10 46 31" src="https://github.com/Parsely/parsely-android/assets/5845095/abc3ffa6-5ebf-4c78-9bdc-79ab7e8e2d16">

That's why I explicitly added `@NonNull` annotation.

Reply:
Praise (❤️): Kudos on making this change, it removes lots of unnecessary nullability checks! 🥇